<launch>

    <param name="/use_sim_time" value="true"/>


    <!-- To un-compress image topics -->
    <node name="republish1" type="republish" pkg="image_transport" output="screen" args="compressed in:=/stereorig/left/image_raw raw out:=/cam0/image_raw" />
    <node name="republish2" type="republish" pkg="image_transport" output="screen" args="compressed in:=/stereorig/right/image_raw raw out:=/cam1/image_raw" />


    <node name="okvis_node" pkg="okvis_ros" type="okvis_node">

        <param name="config_filename" value="$(find okvis_ros)/okvis/config/config_flir_stereo.yaml" />
        <param name="mesh_file" value="firefly.dae" />


        <remap from="/camera0" to="/cam0/image_raw"/>
        <remap from="/camera1" to="/cam1/image_raw"/>

        <remap from="/imu" to="/imu/imu" />

    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find okvis_ros)/config/rviz.rviz"/>


</launch>



